### PR TITLE
Languagetool, and quick word structure.

### LINKED ISSUE #20: docs: languagetool it
I'll throw LT premium at it to see what I missed.

### PR DESCRIPTION
**Pull request by jtagcat**
Closes #20 

A larger overhaul needs to be completed, but that needs more time, and I don't have the time for that. https://github.com/jtagcat/jtagcat

It seems, that I'll be abandoning activity in overnode indefinitely.

### CODE REVIEW

**Comment by jtagcat** [comment]
Closes #20

**Comment by avkonst** [comment]
Thanks. I will have a look.

File: docs/docs/04-workflow-essentials.md [hunk 1]
```diff
@@ -50,25 +50,25 @@ We need to create Docker Compose file, which defines the service. Check out the
...
 version: "3.7"
 services:
-    echo:
-        # Use bridge mode to attach the container to the cluster network
-        network_mode: bridge
-        image: ealen/echo-server
-        restart: unless-stopped
-        ports:
-            - 3000:80
+echo:
```
Comment (avkonst) [inline docs/docs/04-workflow-essentials.md:53]: removed indentation is incorrect

File: docs/docs/04-workflow-essentials.md [hunk 2]
```diff
@@ -50,25 +50,25 @@ We need to create Docker Compose file, which defines the service. Check out the
...
-And the `restart` is set to `unless-stopped` to make it automatically restartable on a crash or host restart. This will make it running like a service but not as one-off process.
+And the `restart` is set to `unless-stopped`, making it automatically restart on a crash or host restart. This will make it running like a service but not as one-off process.
 :::
 
 We save it to `echo/service.yml` file and reference it in the [overnode.yml](overnode-yml-file-description) file by adding the following section:
 
 ```yml
 echo:
-    echo/service.yml: *
+echo/service.yml: *
```
Comment (avkonst) [inline docs/docs/04-workflow-essentials.md:71]: same

File: docs/docs/05-service-discovery.md [hunk 3]
```diff
@@ -13,12 +13,12 @@ It is possible to configure a different name using `hostname` field for a servic
 ```yml
 version: "3.7"
 services:
-    echo:
-        # change default 'echo.weave.local' to the alternative:
-        hostname: echo-server.weave.local
-        network_mode: bridge
-        restart: unless-stopped
-        image: ealen/echo-server
+echo:
```
Comment (avkonst) [inline docs/docs/05-service-discovery.md:16]: same

File: docs/docs/05-service-discovery.md [hunk 4]
```diff
@@ -30,15 +30,15 @@ In general, it is not required to explicitly configure DNS server IP address for
...
-    haproxy:
-        image: haproxy:alpine
-        network_mode: bridge
-        init: true
-        ports:
-            - 443:443
-        restart: unless-stopped
-        environment:
-            - NAMESERVER=${OVERNODE_BRIDGE_IP}:53
+haproxy:
```
Comment (avkonst) [inline docs/docs/05-service-discovery.md:33]: same

File: docs/docs/05-service-discovery.md [hunk 5]
```diff
@@ -49,34 +49,33 @@ If the same service is placed for multiple nodes and you would like to have uniq
 ```yml
 version: "3.7"
 services:
-    echo:
-        # container on the node 1 will have 'echo-1.weave.local' hostname
-        hostname: echo-${OVERNODE_ID}.weave.local
-        network_mode: bridge
-        restart: unless-stopped
-        image: ealen/echo-server
+echo:
```
Comment (avkonst) [inline docs/docs/05-service-discovery.md:52]: same

**Review by avkonst — CHANGES_REQUESTED** [review]
Wording changes are good. Fixes to indentations are not good.

**Comment by jtagcat** [comment]
todo: 6, 7, 0, 21

**Comment by jtagcat** [comment]
Indentation issues are gone. Good notice, My tools didn't show any diffs at the indentation parts, as no text was technically changed.

**Pull request merged**